routes/web/webhook/github.rb

```diff
@@ -68,6 +68,10 @@ def handle_workflow_job(data)
       return error("Unmatched label")
     end
 
+    if Github.runner_labels[label]["gpu"] && !installation.project.get_ff_enable_gpu_runners
```

Comment:
`enable_gpu_runners` feature flag's added in the next commit. Just be sure you deploy them together.